graph-manager-docs/source/getting-started.mdx

```diff
@@ -114,7 +114,7 @@ To begin pushing these traces to Graph Manager, read [Pushing trace data to Grap
 
 Your schema is now registered with Graph Manager, and your server is pushing operation traces!
 
-However, to get the most out of Graph Manager, you need to _re_-register your server's schema every time it changes. The easiest way to do this is to make schema registration part of every deployment in your application's continuous delivery flow. [See an example.](schema-registry/#registering-a-schema-via-continuous-delivery)
+However, to get the most out of Graph Manager, you need to _re_-register your server's schema every time it changes. The easiest way to do this is to make schema registration part of every deployment in your application's continuous delivery flow. [See an example.](schema-registry/#registering-with-continuous-delivery)
```

Comment:
we should probably recommend using schema reporting for this instead of pointing to CD

Reply:
Agreed. I'd prefer to leave out of this PR, as there is a separate set of changes coming for this article that will bring schema reporting much more front and center.